integration_tests/steps/kpi.py

```diff
@@ -0,0 +1,38 @@
+"""
+Copyright 2018 Grid Singularity
+This file is part of D3A.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+# from behave import then
+# from math import isclose
+# from pendulum import duration
+# from d3a_interface.constants_limits import ConstSettings
+# from d3a.constants import DEFAULT_PRECISION
+import os
+import json
+import glob
+import ast
+from behave import then
+
+
+@then('the export functionality of kpi of {areas} are correctly reported')
+def test_export_of_kpi_result(context, areas):
+    sim_data_csv = glob.glob(os.path.join(context.export_path, "*",
+                                          "aggregated_results", "KPI.json"))
+    with open(sim_data_csv[0], "r") as sf:
+        kpi_data = json.load(sf)
+    areas = ast.literal_eval(areas)
```

Comment:
It should work without this additional parsing, schouldn't it?

Reply:
Its an string object, so had to convert it to dict

Reply:
Hmm, doesnt deal `json.load()` with that already?

Reply:
`json.load()` is used for `kpi_data` which is an actual kpi after running the simulation, whereas `areas` variable contains the expected kpi

Reply:
ok, resolved offline

Reply:
@faizan2590 but you could rename areas to "expected_kpis" in order to make it more readable (and to prevent this confusion).

Reply:
Doing it so!